specs/pvs/corla.pvs

```diff
@@ -2800,23 +2803,112 @@ BEGIN
+%%% the endpoint callback (the various \texttt{dashboard} theories)
+%%% and we name the semantially relevant fields using a list of
+%%% strings using the \texttt{json\_spec} type.
   json_spec: TYPE = list[string]
+%%% The \texttt{nothing} constant specifies that no content is
+%%% transmitted for a given communication with the server. This
+%%% commonly happens, e.g., when performing a \texttt{GET} on an
+%%% endpoint.
   nothing: json_spec = null
-%%% This is the standard response that the server gives to all public
-%%% inbound events to permit the UI (the view of the MVC architecture)
-%%% to know what state the server is in, which UI elements should be
-%%% enabled (and thus what transitions are legal).
+%%% The \texttt{server\_response} concept is the standard/normal
+%%% response that the server gives to all public inbound events to
+%%% permit the UI (the view of the MVC architecture) to know what
+%%% state the server is in, which UI elements should be enabled (and
+%%% thus what transitions are legal).  We call this kind of behavior
+%%% in the server ``normal behavior''. In all normal behavior cases
+%%% the server will answer with an HTTP response code of
+%%% \texttt{OK\_200}.
   server_response: json_spec =
     (: "server_state", "ui_events" :)
-% Note that this type is used as a response for both ballot manifest
-% and CVR uploads.
+%%% The \texttt{upload\_response} concept is used to encode the
+%%% variety of response status the server can give to an upload. Note
+%%% that the \texttt{upload\_response} type is used as a response for
+%%% both ballot manifest and CVR uploads. These responses can happen
+%%% synchronously (our plan for \emph{stage-1} deliverable) or
+%%% asynchronously (our plan for a later deliverable). Asynchronous
+%%% updates happen via a call to the \texttt{refresh...} endpoints.
   upload_response: json_spec =
-    % legal values for status are:
-    % "too_late", "data_transmission_interrupted", "upload_sucessful",
-    % "checking_hash", "hash_wrong", "hash_verified", "parsing_data",
-    % "file_type_wrong", "data_parsed"
-    (: "status" :)
+%%% Legal values for \texttt{status} are: \texttt{too\_late},
+%%% \texttt{data\_\-transmission\_\-interrupted},
+%%% \texttt{upload\_\-sucessful}, \texttt{checking\_hash},
+%%% \texttt{hash\_wrong}, \texttt{hash\_verified},
+%%% \texttt{parsing\_\-data}, \texttt{file\_\-type\_\-wrong}, and
+%%% \texttt{data\_parsed}.
+    (: "ballot_manifest-status", "cvr-status" :)
+%%% There are several possible cases where a client can make an
+%%% inappropriate call to the server. Our client will be guarnateed to
+%%% never make sure a call, but we cannot presume that only our client
+%%% will be communicating with the server, as the server will be
+%%% publicly accessible on the internet. We call these behaviors of
+%%% the server ``exceptional behavior''. All responses to exceptional
+%%% behavior come in the form of HTTP client error (\texttt{3xx}) or
```

Comment:
You accidentally subtracted 100 from each of of the status code ranges: `3xx` status codes are for redirection; `4xx` are client errors, `5xx` are server.